### ISSUE TITLE
Can't manipulate some of components in ArenaLive3

### ISSUE THREAD

**Issue by sasadubi** [Schaka/WotLK-AddOns_TBC#2]
Hello there!

There are some components that cannot be manipulated in ArenaLive3 settings. These components are:
- Target of Party member
- Cast bar of Party member

Whatever you do in settings, it just goes back to default, and no changes are applied to these 2 components.

![ControlsNotApplicable](https://github.com/user-attachments/assets/07b04683-d623-48dd-ac8c-99349bae9431)
![cantBeEdited](https://github.com/user-attachments/assets/b1b3ff88-9d9f-4e8b-b2c8-eea14b8b8034)

Thankfully in advance

**Comment by Schaka** [comment]
Been trying to reproduce this and really can't. Are you sure you aren't using some conflicting addons?

**Comment by sasadubi** [comment]
@Schaka Sorry, totally missed out your question about conflicting addons.
What could be conflicting addons, because I don't get anything in BugSack/BugGrabber that indicates conflict with ArenaLive?
I'll try to disable ALL addons except ArenaLive and test it out again.

**Comment by Schaka** [comment]
Yes, please try. I tried it on both the Wrath and MoP versions I have of it and nothing reset.
In what way are settings not saved? Maybe you have general corruption going on for ArenaLive files?

Back up your WTF, start over - that may already be it

**Comment by sasadubi** [comment]
@Schaka 
Tried it again, caught this error in BugGrabber:

![Image](https://github.com/user-attachments/assets/d05d3a35-1b11-40a1-99c4-86ab57f95ad4)


Also this from BugSack:
1x ArenaLive3-3.2.0b\CoreOption.lua:280: attempt to call method 'IsEnabled' (a nil value)
<in C code>: in function `Hide'
Interface\FrameXML\UIParent.lua:1564: in function `SetUIPanel':
Interface\FrameXML\UIParent.lua:1591: in function `MoveUIPanel':
Interface\FrameXML\UIParent.lua:1622: in function `HideUIPanel':
Interface\FrameXML\UIParent.lua:1315: in function <Interface\FrameXML\UIParent.lua:1307>:
<in C code>: in function `SetAttribute'
Interface\FrameXML\UIParent.lua:1990: in function `HideUIPanel':
Interface\FrameXML\UIParent.lua:2141: in function <Interface\FrameXML\UIParent.lua:2121>:
<in C code>: in function `securecall'
Interface\FrameXML\UIParent.lua:1381: in function `ShowUIPanel':
Interface\FrameXML\UIParent.lua:1311: in function <Interface\FrameXML\UIParent.lua:1307>:
<in C code>: in function `SetAttribute'
Interface\FrameXML\UIParent.lua:1974: in function `ShowUIPanel':
BugSack-3.3\BugSack.lua:352: in function <BugSack\BugSack.lua:334>
BugSack-3.3\BugSack.lua:421: in function `OpenSack'
BugSack-3.3\BugSackLDB.lua:46: in function `OnClick'
LibDBIcon-1.0-21 (WeakAuras):146: in function <...ddOns\WeakAuras\Libs\LibDBIcon-1.0\LibDBIcon-1.0.lua:146>

Locals:
(*temporary) = InterfaceOptionsFrame {
 0 = <userdata>
 numTabs = 2
 selectedTab = 2
 lastFrame = GameMenuFrame {}
}
(*temporary) = <unnamed> {
 ShowUIPanelFailed = <function> @ Interface\FrameXML\UIParent.lua:1515:
 0 = <userdata>
 UpdateUIPanelPositions = <function> @ Interface\FrameXML\UIParent.lua:1654:
 HideUIPanel = <function> @ Interface\FrameXML\UIParent.lua:1602:
 UIParentManageFramePositions = <function> @ Interface\FrameXML\UIParent.lua:1747:
 MoveUIPanel = <function> @ Interface\FrameXML\UIParent.lua:1586:
 SetUIPanel = <function> @ Interface\FrameXML\UIParent.lua:1522:
 ShowUIPanel = <function> @ Interface\FrameXML\UIParent.lua:1327:
 GetUIPanel = <function> @ Interface\FrameXML\UIParent.lua:1646:
}
(*temporary) = <function> defined =[C]:-1

  ---



and


1x ArenaLive3-3.2.0b\CoreOption.lua:280: attempt to call method 'IsEnabled' (a nil value)
<in C code>: in function `Hide'
Interface\FrameXML\InterfaceOptionsFrame.lua:202: in function `InterfaceOptionsFrame_Show':
Interface\FrameXML\InterfaceOptionsFrame.lua:244: in function <Interface\FrameXML\InterfaceOptionsFrame.lua:224>:

Locals:
(*temporary) = InterfaceOptionsFrame {
 0 = <userdata>
 numTabs = 2
 selectedTab = 2
 lastFrame = GameMenuFrame {}
}
(*temporary) = InterfaceOptionsFrameOkay {
 0 = <userdata>
}
(*temporary) = <function> defined =[C]:-1

  ---

**Comment by Schaka** [comment]
That seems like some conflict/corruption. Are you using ClassicAPI?
Did you try without any addons besides ArenaLive Core + Unitframes?

It seems that Bugsack and WeakAuras are in the stacktrace

**Comment by sasadubi** [comment]
So for first screenshot and 2 logs that I posted, ClassicAPI was OFF, but WeakAura wasn't.

Now that I turned off, literally everything, except BugGrabber, BugSack and ArenaLive, I get the following:

1x ArenaLive3-3.2.0b\CoreOption.lua:280: attempt to call method 'IsEnabled' (a nil value)
<in C code>: in function `ClearFocus'
<string>:"*:OnEnterPressed":1: in function <[string "*:OnEnterPressed"]:1>

Locals:
(*temporary) = ALUF_UnitFrameOptionsHandlerFrameElementXOffset {
 0 = <userdata>
 OnLeave = <function> @ ArenaLive3\CoreOption.lua:92:
 GetDBValue = <function> @ ArenaLiveUnitFrames3\Options\Position.lua:183:
 decimal = true
 group = "PartyFrames"
 handler = "TargetFrame"
 type = "EditBox"
 OnShow = <function> @ ArenaLive3\CoreOption.lua:98:
 postUpdate = <function> @ ArenaLiveUnitFrames3\Options\Position.lua:185:
 addon = "ArenaLiveUnitFrames3"
 UpdateShownValue = <function> @ ArenaLive3\CoreOption.lua:127:
 SetDBValue = <function> @ ArenaLiveUnitFrames3\Options\Position.lua:184:
 title = ALUF_UnitFrameOptionsHandlerFrameElementXOffsetTitle {}
 OnEditFocusGained = <function> @ ArenaLive3\CoreOption.lua:275:
 OnEnter = <function> @ ArenaLive3\CoreOption.lua:78:
 tooltip = "Horizontal distance between the frame element and the element it is attached to."
 OnEditFocusLost = <function> @ ArenaLive3\CoreOption.lua:279:
}
(*temporary) = ALUF_UnitFrameOptionsHandlerFrameElementXOffset {
 0 = <userdata>
 OnLeave = <function> @ ArenaLive3\CoreOption.lua:92:
 GetDBValue = <function> @ ArenaLiveUnitFrames3\Options\Position.lua:183:
 decimal = true
 group = "PartyFrames"
 handler = "TargetFrame"
 type = "EditBox"
 OnShow = <function> @ ArenaLive3\CoreOption.lua:98:
 postUpdate = <function> @ ArenaLiveUnitFrames3\Options\Position.lua:185:
 addon = "ArenaLiveUnitFrames3"
 UpdateShownValue = <function> @ ArenaLive3\CoreOption.lua:127:
 SetDBValue = <function> @ ArenaLiveUnitFrames3\Options\Position.lua:184:
 title = ALUF_UnitFrameOptionsHandlerFrameElementXOffsetTitle {}
 OnEditFocusGained = <function> @ ArenaLive3\CoreOption.lua:275:
 OnEnter = <function> @ ArenaLive3\CoreOption.lua:78:
 tooltip = "Horizontal distance between the frame element and the element it is attached to."
 OnEditFocusLost = <function> @ ArenaLive3\CoreOption.lua:279:
}
(*temporary) = <function> defined =[C]:-1

  ---

I think I got ClassicAPI from Evolve's addon pack like 2 weeks ago.

**Comment by sasadubi** [comment]
1x ArenaLive3-3.2.0b\CoreOption.lua:280: attempt to call method 'IsEnabled' (a nil value)
<in C code>: in function `Hide'
Interface\FrameXML\InterfaceOptionsFrame.lua:202: in function `InterfaceOptionsFrame_Show':
Interface\FrameXML\InterfaceOptionsFrame.lua:244: in function <Interface\FrameXML\InterfaceOptionsFrame.lua:224>:

Locals:
(*temporary) = InterfaceOptionsFrame {
 0 = <userdata>
 numTabs = 2
 selectedTab = 2
 lastFrame = GameMenuFrame {}
}
(*temporary) = InterfaceOptionsFrameOkay {
 0 = <userdata>
}
(*temporary) = <function> defined =[C]:-1

  ---

**Comment by Schaka** [comment]
This is way more helpful, thank you. I'll try and reproduce it and if I can't, this may give me enough to try and fix things blindly

**Comment by sasadubi** [comment]
@Schaka Thanks man, much appreciated!
Let me know when you need me to test something.
